### PR TITLE
Add Turkish News Category Dataset - 270K - Lite Version

### PR DESCRIPTION
**Pull request by yavuzKomecoglu**
This PR adds the Turkish News Categories Dataset (270K - Lite Version) dataset which is a text classification dataset by me, @basakbuluz and @serdarakyol.
This dataset contains the same news from the current [interpress_news_category_tr dataset](https://huggingface.co/datasets/interpress_news_category_tr) but contains less information, OCR errors are reduced, can be easily separated, and can be divided into 10 classes ("kültürsanat", "ekonomi", "siyaset", "eğitim", "dünya", "spor", "teknoloji", "magazin", "sağlık", "gündem") were rearranged.

### CODE REVIEW

File: datasets/interpress_news_category_tr_lite/README.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+}
+```
+
+### Data Fields
+
+- **category** : Indicates to which category the news text belongs.
+(Such as "kültürsanat", "ekonomi", "siyaset", "eğitim", "dünya", "spor", "teknoloji", "magazin", "sağlık", "gündem")
```
Comment (lhoestq) [inline datasets/interpress_news_category_tr_lite/README.md:78]:
```suggestion
(Such as "kültürsanat" (0), "ekonomi" (1), "siyaset" (2), "eğitim" (3), "dünya" (4), "spor" (5), "teknoloji" (6), "magazin" (7), "sağlık" (8), "gündem" (9))
```
we can show the mapping between the label ids (integers) and the label names (strings)

File: datasets/interpress_news_category_tr_lite/README.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+languages:
+- tr
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 100k<n<1M
+source_datasets:
+- original
```
Comment (lhoestq) [inline datasets/interpress_news_category_tr_lite/README.md:15]:
```suggestion
- extended|interpress_news_category_tr
```
This dataset is based on the interpress_news_category_tr one right ?

**Review by lhoestq — APPROVED** [review]
Awesome thanks :)

File: datasets/interpress_news_category_tr_lite/README.md (hunk 2 above)
Comment (yavuzKomecoglu) [reply datasets/interpress_news_category_tr_lite/README.md]: True

**Comment by lhoestq** [comment]
Thanks for the change, merging now !

**Pull request merged**